src/api/config/application.rb

```diff
@@ -87,6 +87,12 @@ class Application < Rails::Application
 
     config.action_controller.perform_caching = true
 
+    # FIXME: Check all the places in the codebase where we cache content
```

Comment:
@hennevogel so maybe not marking as fixme?

Reply:
yeah just disable it! :)